test/core/utils/is-hidden.js

```diff
@@ -1,3 +1,19 @@
+function createContentHidden() {
+	'use strict';
+	var group = document.createElement('div');
+	group.innerHTML = '<label id="mylabel">Label</label><input aria-labelledby="mylabel" type="text" />';
+	return group;
+}
+
+function makeShadowTreeHidden(node) {
```

Comment:
For the shadow DOM test functions, I am using a convention of adding something unique to the end of the makeShadowTree - I used "hidden" because this is the test file for  the `isHidden` function and it seemed like overkill to have to write `makeShadowTreeIsHidden` - its just a test function though, so I don't think its a big deal